salt/modules/disk.py

```diff
@@ -481,12 +481,13 @@ def fstype(device):
                 fs_type = df_out[2]
                 if fs_type:
                     return fs_type
-        else:
-            df_out = __salt__['cmd.run']('df -T {0}'.format(device)).splitlines()
-            if len(df_out) > 1:
-                fs_type = df_out[1]
-                if fs_type:
-                    return fs_type
```

Comment:
I will leave it up to Mike, but IMO it is better to have both, then for `file -s` to not work on some obscure distribution.

Also, you have some trailing whitespaces https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/9364/console